package.json

```diff
@@ -16,7 +17,7 @@
     "start": "npm run lerna run start",
     "bootstrap": "npm run lerna bootstrap --hoist",
     "postinstall": "npm run bootstrap",
-    "test": "bash scripts/test.sh",
+    "test": "./node_modules/.bin/jest",
```

Comment:
You could only use following line in this part
```
"test": "jest"
```

Reply:
That does work, but if jest is installed globally, the global version will be used instead of the local one.  That opens us up to versioning issues.  That's why I always use `./node_modules/.bin`.